pages/linux/snap.md

```diff
@@ -0,0 +1,24 @@
+# snap
+
+> Tool for managing the *snap* self-contained softwawre packages.
+> Similar to what *apt* is for *.deb*.
+
+- Look for an app:
+
+`snap find {{package}}`
+
+- Install software:
+
+`snap install {{package}}`
+
+- Find installed *snap* software:
+
+`snap list`
+
+- Remove programs:
```

Comment:
Perhaps `packages` would increase clarity here?

Reply:
apps or packages sound good to me.

Reply:
Took into consideration the received comments and submitted it again, please give it look. Thanks!